package.json

```diff
@@ -147,7 +135,18 @@
     "whatwg-fetch": "1.1.1"
   },
   "devDependencies": {
-    "babel-register": "6.26.0",
+    "@babel/core": "^7.0.0",
```

Comment:
@brbrr Any concerns merging these with `ˆ` range at first and let Renovate bot pin dependencies later on to exact versions?

Reply:
We can pin the dependencies to the exact latest versions, if that's the Jetpack custom.

Reply:
Follow up https://github.com/Automattic/jetpack/pull/11689